MozillaFoundation/foundation.mozilla.org
# Main Nav Fake Data Factory 2.0

# Description

This corrected PR introduces a fake data factory for the main navigation.

The factory should mimic fairly closely as far as number and types of items. Like any fake data factory, there is a lot of fake generated links/pages/content, but some of the nav is real in order to more accurately capture the essence of the main nav on production. The factory is granular enough that we can specify any real content that might be helpful to have.

If there's anything specific you'd like to have linked within the factory instead of fake data, please let me know and I can adjust.

Related PRs/issues: TP-353 #12131 #12571

# To Test

1) Check out this branch locally
2) Run `inv new-db` to get a new database
3) Note the autogenerated fake main navigation
4) Log in locally via the /cms route
5) Verify there are now no routing errors

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-910)
